.github/workflows/mvn-verify-check.yml

```diff
@@ -24,12 +24,13 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  get-noSnapshot-versions-from-dist:
+  get-all-versions-from-dist:
```

Comment:
Please suggest a better name if this doesn't make sense.

Reply:
maybe just neutral `get-shim-versions-from-dist` since we want to remain flexible to extract different version subsets for different purposes. Not super important as long as it works and has enough comments for each use case